assets/css/template.css

```diff
@@ -0,0 +1,537 @@
+    list-style: none;
+    z-index: 3;
+    display: inline;
+}
+#pelagos-menu-1 ul.menu ul {
+    padding: 0;
+    list-style: none;
+    background: #004250;
+}
+#pelagos-menu-1 ul.menu li ul {
+    position: absolute;
+    width: 230px;
+    left: -999em;
+}
+#pelagos-menu-1 ul.menu li li {
+    float: left;
+    width: 230px;
+}
+#pelagos-menu-1 ul.menu li li a {
+    text-transform: capitalize;
+}
+
+#pelagos-menu-1 ul.menu li li span:hover,
+#pelagos-menu-1 ul.menu li li span:active,
+#pelagos-menu-1 ul.menu li li a:active,
+#pelagos-menu-1 ul.menu li li a:hover {
+    background: #c2deea;
+    text-decoration: none;
+}
+
+#pelagos-menu-1 ul.menu li li.expanded ul {
+    margin: -25px 0 0 230px;
+}
+
+#pelagos-menu-1 a:active, #pelagos-menu-1 span.nolink:active,
+#pelagos-menu-1 ul.menu li li.expanded ul.menu a,
+#pelagos-menu-1 ul.menu li li.expanded ul.menu span {
+    background: #004250;
+}
+
+#pelagos-menu-1 ul.menu li li.expanded ul.menu span:active,
+#pelagos-menu-1 ul.menu li li.expanded ul.menu span:hover,
+#pelagos-menu-1 ul.menu li li.expanded ul.menu a:hover,
+#pelagos-menu-1 ul.menu li li.expanded ul.menu a:active,
+#pelagos-menu-1 a:hover, #pelagos-menu-1 span.nolink:hover {
+    background: #c2deea;
+}
+
+#pelagos-menu-1 ul.menu li li.expanded ul.menu li.expanded a,
+#pelagos-menu-1 ul.menu li li.expanded ul.menu li.expanded span,
+#pelagos-menu-1 ul.menu li li.expanded a,
+#pelagos-menu-1 ul.menu li li.expanded span {
+    background: #004250 url("/build/images/menu_arrow.png") 100% 50% no-repeat;
+}
+#pelagos-menu-1 ul.menu li li.expanded ul.menu li.expanded a:active,
+#pelagos-menu-1 ul.menu li li.expanded ul.menu li.expanded a:hover,
+#pelagos-menu-1 ul.menu li li.expanded ul.menu li.expanded span:active,
+#pelagos-menu-1 ul.menu li li.expanded ul.menu li.expanded span:hover,
+#pelagos-menu-1 ul.menu li li.expanded span:active,
+#pelagos-menu-1 ul.menu li li.expanded span:hover,
+#pelagos-menu-1 ul.menu li li.expanded a:active,
+#pelagos-menu-1 ul.menu li li.expanded a:hover {
+    background: #c2deea url("/build/images/menu_arrow.png") 100% 50% no-repeat;
+}
+
+#pelagos-menu-1 ul.menu li li li:hover ul,
+#pelagos-menu-1 ul.menu li li li.sfhover ul,
+#pelagos-menu-1 ul.menu li li:hover ul,
+#pelagos-menu-1 ul.menu li li.sfhover ul {
+    left: auto;
+}
+
+#pelagos-menu-1 ul.menu li:hover ul ul,
+#pelagos-menu-1 ul.menu li:hover ul ul ul,
+#pelagos-menu-1 ul.menu li.sfhover ul ul,
+#pelagos-menu-1 ul.menu li.sfhover ul ul ul {
+    left: -999em;
+}
+
+/* Jira buttons */
+.atlassian-btns {
+    z-index: 10000;
+    position: fixed;
+    width: 500px;
+    height: 50px;
+    top: 0%;
+    left: 50%;
+    margin-top: -2px;
+    margin-left: -122px;
+}
+
+.atlassian-btns li {
+    display: inline-block;
+}
+
+.atlassian-btns li a {
+    color: white !important;
+    font-size: 9pt;
+    font-family: sans-serif;
+    font-weight: normal;
+    padding: 0 5px 0 5px;
+    border-width: 0px 1px 1px;
+    border-top-style: initial;
```

Comment:
If all these are set the same, you can use border for all the sides (left, right, bottom etc...)